source/install.html.haml

```diff
@@ -44,20 +44,33 @@ no_container: true
       There are also community-maintained wrappers for the following languages:
 
     %ul
-      %li
-        #{link_to 'Java', 'https://mvnrepository.com/artifact/de.larsgrefer.sass'},
-        including
-        #{link_to 'Gradle', 'https://docs.freefair.io/gradle-plugins/current/reference/#_embedded_sass'}
-        and
-        #{link_to 'Maven', 'https://github.com/HebiRobotics/sass-cli-maven-plugin'}
-        plugins.
-
       %li
         #{link_to 'Ruby', 'https://github.com/ntkme/sass-embedded-host-ruby#readme'}
 
       %li
         #{link_to 'Swift', 'https://github.com/johnfairh/swift-sass#readme'}
 
+      %li
+        #{link_to 'Java', 'https://mvnrepository.com/artifact/de.larsgrefer.sass'},
```

Comment:
I moved the links to Java to the bottom because with the sublist it kinda obscured the other two options. They are no longer in alphabetical order, not sure how much that matters in this case.